models/instance/fact_instance_activity.sql

```diff
@@ -0,0 +1,17 @@
+{{
+    config(
+        materialized="materialized_view",
+        schema=env_var("ASPECTS_XAPI_DATABASE", "xapi"),
+        engine=get_engine("SummingMergeTree()"),
```

Comment:
I think this is our first use of a SummingMergeTree. I'm still trying to confirm this is all working as expected.

Reply:
Were you able to confirm that this does what we want? It looks correct to me but I'm trying to test this against the `AggregatingMergeTree` table engine.

Reply:
There's something off with it when there is new data coming in, I'm still debugging it